tests/FusionAuth/FusionAuthClientTest.php

```diff
@@ -1,176 +1,209 @@
+    {
+        // Create it
+        $response = $this->client->createApplication(null, ["application" => ["name" => "PHP Client Application"]]);
+        $this->handleResponse($response);
+        $this->applicationId = $response->successResponse->application->id;
+
+        // Retrieve it
+        $response = $this->client->retrieveApplication($this->applicationId);
+        $this->handleResponse($response);
+        $this->assertEquals("PHP Client Application", $response->successResponse->application->name);
+
+        // Update it
+        $response = $this->client->updateApplication(
+            $this->applicationId,
+            ["application" => ["name" => "PHP Client Application Updated"]]
+        );
+        $this->handleResponse($response);
+        $this->assertEquals("PHP Client Application Updated", $response->successResponse->application->name);
+
+        // Retrieve it again
+        $response = $this->client->retrieveApplication($this->applicationId);
+        $this->handleResponse($response);
+        $this->assertEquals("PHP Client Application Updated", $response->successResponse->application->name);
+
+        // Deactivate it
+        $response = $this->client->deactivateApplication($this->applicationId);
+        $this->handleResponse($response);
+
+        // Retrieve it again
+        $response = $this->client->retrieveApplication($this->applicationId);
+        $this->handleResponse($response);
+        $this->assertFalse($response->successResponse->application->active);
+
+        // Retrieve inactive
+        $response = $this->client->retrieveInactiveApplications();
+        $this->assertEquals("PHP Client Application Updated", $response->successResponse->applications[0]->name);
+
+        // Reactivate it
+        $response = $this->client->reactivateApplication($this->applicationId);
+        $this->handleResponse($response);
+
+        // Retrieve it again
+        $response = $this->client->retrieveApplication($this->applicationId);
+        $this->handleResponse($response);
+        $this->assertEquals("PHP Client Application Updated", $response->successResponse->application->name);
+        $this->assertTrue($response->successResponse->application->active);
+
+        // Delete it
+        $response = $this->client->deleteApplication($this->applicationId);
+        $this->handleResponse($response);
+
+        // Retrieve it again
+        $response = $this->client->retrieveApplication($this->applicationId);
+        $this->assertEquals(404, $response->status);
+
+        // Retrieve inactive
+        $response = $this->client->retrieveInactiveApplications();
+        $this->assertEmpty($response->successResponse->applications);
     }
 
-    $this->assertTrue($response->wasSuccessful(), "Expected success. Status: " . $response->status);
-  }
-}
\ No newline at end of file
+    /**
+     * @throws \Exception
+     */
+    public function testCanHandleUsers(): void
```

Comment:
Can you elaborate? Do you want me to write something like "In order to make sure this library runs well, we're testing if it can create/retrieve application/users"?